examples/react/react-native-app/metro-v71.config.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Old custom configuration for React Native v0.71.
```

Comment:
any idea whats going on here?

also seems like we probably need to check that RN app still works

Reply:
yea will check.